cerializer/prepare.pyx

```diff
@@ -0,0 +1,309 @@
+#cython: language_level=3
```

Comment:
This is largely taken from the fastavro, right? You should write it here in module docstring. Add here also the link to the original file on the github and fastavro version used.